cli/tests/test_concurrent_operations.rs

```diff
@@ -55,15 +55,15 @@ fn test_concurrent_operations_auto_rebase() {
     test_env.jj_cmd_ok(&repo_path, &["describe", "-m", "initial"]);
     let stdout = test_env.jj_cmd_success(&repo_path, &["op", "log"]);
     insta::assert_snapshot!(stdout, @r###"
-    @  cfc96ff553b9 test-username@host.example.com 2001-02-03 04:05:08.000 +07:00 - 2001-02-03 04:05:08.000 +07:00
+    @  8626e1f87784 test-username@host.example.com 2001-02-03 04:05:08.000 +07:00 - 2001-02-03 04:05:08.000 +07:00
```

Comment:
Why did the hashes change? Is it the issue with empty remotes you mentioned?

Reply:
New content hash is calculated from `local_branches: BTreeMap<String, RefTarget>` + `remote_views: BTreeMap<String, RemoteView>` + ..., so lengths and contents order differ.

Reply:
Oh, right, the content hash isn't determined by the storage format since a while ago. I had forgotten that.